official/resnet/keras/keras_imagenet_main.py

```diff
@@ -84,7 +84,6 @@ def parse_record_keras(raw_record, is_training, dtype):
       is_training=is_training)
 
   image = tf.cast(image, dtype)
-  label = tf.sparse_to_dense(label, (imagenet_main._NUM_CLASSES,), 1)
```

Comment:
Don't we need to remove this for the synthetic_input_fn as well?

Reply:
Done.